main.py

```diff
@@ -294,6 +294,10 @@ def start(self, settings):
 				self._qmp = QEMUMonitorProtocol(('localhost', 4444))
 				self._qmp.connect()
 			except Exception as e:
+				# Ff XQEMU quits, we abort the loop
```

Comment:
"Ff"

Reply:
Oops, I was sure I checked the spelling. Fixed.